records_mover/db/postgres/unloader.py

```diff
@@ -106,6 +110,11 @@ def can_unload_to_scheme(self, scheme: str) -> bool:
         # and requires no scratch buckets.
         return True
 
+    @contextmanager
+    def temporary_unloadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
+        with TemporaryDirectory(prefix='temporary_unloadable_directory_loc') as dirname:
+            yield FilesystemDirectoryUrl(dirname)
```

Comment:
If we're doing a table2table copy between, say, Redshift and BigQuery, and we don't have a temporary bucket on the BigQuery side, the move() logic needs to be able to pull a bucket directory that we can *un*load into.